frameworks/kafka/tests/test_shakedown_strategy.py

```diff
@@ -0,0 +1,263 @@
+import pytest
+
+import sdk_install as install
+import sdk_tasks as tasks
+import sdk_spin as spin
+import sdk_utils as utils
+import sdk_marathon as marathon
+import shakedown
+import dcos
+import dcos.config
+import dcos.http
+
+from tests.test_utils import (
+    DEFAULT_PARTITION_COUNT,
+    DEFAULT_REPLICATION_FACTOR,
+    PACKAGE_NAME,
+    SERVICE_NAME,
+    DEFAULT_BROKER_COUNT,
+    DEFAULT_TOPIC_NAME,
+    EPHEMERAL_TOPIC_NAME,
+    DEFAULT_POD_TYPE,
+    DEFAULT_PLAN_NAME,
+    DEFAULT_PHASE_NAME,
+    DEFAULT_TASK_NAME,
+    DEPLOY_STRATEGY_SERIAL_CANARY,
+    service_cli,
+    broker_count_check,
+    service_plan_wait
+)
+
+
+def setup_module(module):
+    install.uninstall(SERVICE_NAME, PACKAGE_NAME)
+    utils.gc_frameworks()
+    shakedown.install_package(PACKAGE_NAME,
+                              service_name=SERVICE_NAME,
+                              options_json=install.get_package_options(
+                                  additional_options=DEPLOY_STRATEGY_SERIAL_CANARY
+                              ),
+                              wait_for_completion=True)
```

Comment:
Did you consider using sdk_install::install() here instead? Just wanted to point it out.